nextflow_schema.json

```diff
@@ -321,6 +330,18 @@
         },
         "shortread_complexityfilter": {
             "type": "boolean"
+        },
+        "shortread_complexityfilter_entropy": {
+            "type": "number",
+            "default": 0.3
+        },
+        "shortread_complexityfilter_prinseqplusplus_mode": {
+            "type": "string",
+            "default": "entropy"
```

Comment:
Should this be an enum, too, with a fixed set of options?

Reply:
Uups yes. Will correct in the morning 👍